packages/@adobe/spectrum-css-temp/components/barloader/index.css

```diff
@@ -112,10 +112,10 @@ governing permissions and limitations under the License.
 }
 
 [dir='ltr'] .spectrum-BarLoader--indeterminate .spectrum-BarLoader-fill {
-  animation: indeterminate-loop-ltr var(--spectrum-barloader-large-indeterminate-duration) infinite;
+  animation: indeterminate-loop-ltr var(--spectrum-barloader-large-indeterminate-duration, 1s) infinite;
```

Comment:
@devongovett this looks like a lightningcss bug?
I think the animation name is modularized correctly, but then it's not being found in the look up around here? https://github.com/parcel-bundler/lightningcss/blob/master/src/properties/animation.rs#L71

https://lightningcss.dev/playground/index.html#%7B%22minify%22%3Atrue%2C%22customMedia%22%3Atrue%2C%22cssModules%22%3Atrue%2C%22analyzeDependencies%22%3Afalse%2C%22targets%22%3A%7B%22chrome%22%3A6225920%7D%2C%22include%22%3A0%2C%22exclude%22%3A0%2C%22source%22%3A%22%5Cn.sample-1%20%7B%5Cn%20%20animation%3A%20loop%20var(--foo%2C%201s)%20infinite%3B%5Cn%7D%5Cn.sample-2%20%7B%5Cn%20%20animation%3A%20loop%20var(--foo)%20infinite%3B%5Cn%7D%5Cn%5Cn%40keyframes%20loop%20%7B%5Cn%20%20from%20%7B%5Cn%20%20%20%20transform%3A%20translate(10px)%3B%5Cn%20%20%7D%5Cn%20%20to%20%7B%5Cn%20%20%20%20transform%3A%20translate(40px)%3B%5Cn%20%20%7D%5Cn%7D%5Cn%22%2C%22visitorEnabled%22%3Afalse%2C%22visitor%22%3A%22%7B%5Cn%20%20Color(color)%20%7B%5Cn%20%20%20%20if%20(color.type%20%3D%3D%3D%20'rgb')%20%7B%5Cn%20%20%20%20%20%20color.g%20%3D%200%3B%5Cn%20%20%20%20%20%20return%20color%3B%5Cn%20%20%20%20%7D%5Cn%20%20%7D%5Cn%7D%22%2C%22unusedSymbols%22%3A%5B%5D%2C%22version%22%3A%22local%22%7D

Reply:
Maybe better to switch to separate properties for now instead of the shorthand?

```css
animation-name: infinite-loop-ltr;
```

Then you could combine the other values that are the same into a separate non-direction specific rule:

```css
.spectrum-BarLoader--indeterminate .spectrum-BarLoader-fill {
  animation-duration: var(--spectrum-barloader-large-indeterminate-duration);
  animation-repeat-count: infinite;
}
```

Reply:
@devongovett's suggestion worked great.